npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -27,7 +27,8 @@ var (
 	// ErrUnsupportedExceptCIDR is returned when Except CIDR block translation feature is used in windows.
 	ErrUnsupportedExceptCIDR = errors.New("unsupported Except CIDR block translation features used on windows")
 	// ErrUnsupportedSCTP is returned when SCTP protocol is used in windows.
-	ErrUnsupportedSCTP = errors.New("unsupported SCTP protocol used on windows")
+	ErrUnsupportedSCTP    = errors.New("unsupported SCTP protocol used on windows")
+	ErrInvalidIPV6Address = errors.New("invalid IPV6 address")
```

Comment:
nitpick: seems this error will be thrown for any IPv6 or for an invalid IPv4. Maybe "unsupported" is a better word for IPv6?

Reply:
updated